plugin.test.ts

```diff
@@ -60,6 +60,43 @@ describe("handles quotes around inline code", async () => {
   });
 });
 
+describe("handles quotes at the edges of a paragraph", () => {
+  it("at start after another paragraph", async () => {
+    const file = await process('paragraph\n\n"after paragraph"');
+    expect(file.toString()).toMatchInlineSnapshot(`
+      "paragraph\n\n“after paragraph”
+      "`);
+  });
+
+  it("at start after a blockquote", async () => {
+    const file = await process('> blockquote\n\n"after blockquote"');
+    expect(file.toString()).toMatchInlineSnapshot(`
+      "> blockquote\n\n“after blockquote”
+      "`);
+  });
+
+  it("at start within a blockquote", async () => {
+    const file = await process('> blockquote\n>\n> "within blockquote"');
+    expect(file.toString()).toMatchInlineSnapshot(`
+      "> blockquote\n>\n> “within blockquote”
+      "`);
+  });
+
+  it("at end before another paragraph", async () => {
```

Comment:
The ending quotes actually work fine without this change, but we might as well test them while we're at it!